src/helpers/rendering.ts

```diff
@@ -16,7 +16,6 @@ export function drawHighlight(
   const color = highlight.color || HIGHLIGHT_COLOR;
 
   const { ctx } = renderingContext;
-  ctx.save();
```

Comment:
why is this removed ?

Reply:
because it was added in the reverted commit, but I don't remember why :man_shrugging: 
It was probably a "bug" that only appeared in this case. I'll restore it